src/components/Scrollable/Scrollable.scss

```diff
@@ -1,4 +1,8 @@
 .scrollbar {
+  --scrollable-track-thickness: 12px;
+  --scrollable-thumb-thickness: Calc(var(--scrollable-track-thickness)/2);
```

Comment:
This variable seems to be conflicting with the one below. The thumb's thickness is only half that of the track if the padding is a quarter that of the track.

For example, if the user sets `--scrollable-thumb-offset` to `4px`, it will get out of alignment.

Reply:
Yeah, I didn't understand that part on the original code, because the thumb was half the track (12px & 6px) so I thought there was a reason for this relationship. 

But anyway, the conflict you are mentioning existed before also (just hard-coded instead of variables), and it's up to the developer to know basic CSS to understand this, and I guess the developer will see the output and adjust accordingly.

Only now it is easier to manipulate without writing selectors and manually typing `padding:...` but in our examples page the padding was set manually, so it means it's up to the developers to set everything up perfectly

Reply:
OK cool, didn't realize it was part of the original code 😁